### DISCUSSION TITLE
No output

### DISCUSSION THREAD

**Discussion by devilish-crow** [cc-tweaked/CC-Tweaked#1411] [Q&A]
I get no output from my program, no errors, no logs, nothing. This is my code: 

function CUI(m)
local n=1
local l=#m
while true do
term.clear()
term.setCursorPos(1,2)
for i=1, l, 1 do
if i==n then print(i, "["..m[i].."]") else print(i, m[i]) end
end
print("Select a number[arrow up/arrow down]")
local a, b= os.pullEventRaw()
if a == "key" then
if b==200 and n>1 then n=n-1 end
if b==208 and n<=l then n=n+1 end
if b==28 then break end
end
end
term.clear() term.setCursorPos(1,1)
return n
end

**Comment by Lupus590** [comment]
You never call the function.

**Reply by devilish-crow** [reply]
Sorry because my main programming language is not lua, but how do I call the function in cc:tweaked ?

**Reply by SquidDev** [reply]
Calling user-defined functions is just the same as calling any other function (like you do with `term.clear()` or `print()`). So in your case, just do `CUI(xyz)`, though replacing `xyz` with whatever you want to be passing to this function.
